cmd/ci-secret-bootstrap/main_test.go

```diff
@@ -1470,10 +1240,8 @@ func TestConstructSecrets(t *testing.T) {
 			},
 		},
 		{
-			name: "Secret fails when base64 decoding is requsted on invalid data",
-			bwItems: []bitwarden.Item{
-				{Name: "item", Fields: []bitwarden.Field{{Name: "key", Value: "value"}}},
-			},
+			name:  "Secret fails when base64 decoding is requsted on invalid data",
+			items: []testVaultItem{{name: "item", fields: vaultFieldsFromMap(map[string]string{"key": "value"})}},
```

Comment:
In the future, please do not change formatting like this (You did this in almost every single test case and there are a lot). Your reviewer has to review all these formatting changes for no good reason.

Reply:
well this is a completely different field value, but I must admin that I've changed the formating :D